src/query/query_asset_definition.rs

```diff
@@ -13,25 +10,17 @@ use cosmwasm_std::{to_binary, Binary};
 ///
 /// * `deps` A dependencies object provided by the cosmwasm framework.  Allows access to useful
 /// resources like contract internal storage and a querier to retrieve blockchain objects.
-/// * `qualifier` An enum containing identifier information that can be used to look up a stored
-/// [AssetDefinitionV2](crate::core::types::asset_definition::AssetDefinitionV2).
-pub fn query_asset_definition(deps: &DepsC, qualifier: AssetQualifier) -> AssetResult<Binary> {
-    let asset_definition = match qualifier {
-        AssetQualifier::AssetType(asset_type) => {
-            may_load_asset_definition_v2_by_type(deps.storage, asset_type)
-        }
-        AssetQualifier::ScopeSpecAddress(scope_spec_address) => {
-            may_load_asset_definition_v2_by_scope_spec(deps.storage, scope_spec_address)
-        }
-    }?;
+/// * `asset_type` The asset type corresponding to the stored
+/// [AssetDefinitionV2](crate::core::types::asset_definition::AssetDefinitionV2) to look up.
```

Comment:
You will likely need to do a sweep of all comments to swap out `AssetDefinitionV2` with `AssetDefinitionV3`, because otherwise `cargo doc` will probably have an aneurysm :(

Reply:
done, doc-test definitely was complaining, 😄